Client/Frontend/Popup/AlertPopupView.swift

```diff
@@ -48,6 +52,22 @@ class AlertPopupView: PopupView {
         messageLabel.numberOfLines = 0
         containerView.addSubview(messageLabel)
         
+        if let inputType = inputType {
```

Comment:
add `let textField = textField` here to avoid unwrappings

Reply:
Not needed here with change to use Then framework.